tests/auth/test_generic_auth.py

```diff
@@ -250,49 +161,3 @@ def test_parent_domain():
     assert parent_domain("smtp.example.a.com") == parent_domain("imap.a.com")
 
     assert parent_domain("company.co.uk") != parent_domain("evilcompany.co.uk")
-
-
-@pytest.mark.usefixtures("mock_smtp_get_connection")
-def test_successful_reauth_resets_sync_state(db, mock_imapclient):
```

Comment:
Is this not correct anymore?

Reply:
Restored the original behavior in https://github.com/closeio/sync-engine/pull/27/commits/f1c86fd3e1134c3252d26a2e8585b7a3105e0a77